gensim/parsing/preprocessing.py

```diff
@@ -38,77 +38,250 @@
 """
 STOPWORDS = frozenset(w for w in STOPWORDS.split() if w)
 
+RE_PUNCT = re.compile(r'([%s])+' % re.escape(string.punctuation), re.UNICODE)
```

Comment:
No needed empty lines between regexps + please add comment for each regexp `  # remove punctuation according to string.punctuation